plugin/service/host/state.go

```diff
@@ -7,12 +7,10 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"net"
-	"net/http"
-	"time"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	smithyEndpoints "github.com/aws/smithy-go/endpoints"
```

Comment:
not a blocker, but still unclear how smithy is used, is it something inherited from the sdk?

Reply:
The latest aws-sdk-go-v2 client utilizes this for custom endpoint resolvers.